src/commands/mod.rs

```diff
@@ -233,11 +267,24 @@ pub struct EdgeAppManifest {
     pub settings: Vec<Setting>,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Default, EnumString, Display, EnumIter)]
+pub enum SettingType {
+    #[default]
+    #[strum(serialize = "String", to_string = "string")]
```

Comment:
why is it starting with capital letter???

Reply:
to represent enum value name as string 1:1 but I can just make it #[strum(serialize = "string")] and it will work

Reply:
Why do you want to represent it 1:1? Our spec says string/secret. not String/Secret. Please don't change stuff like that.